### PR TITLE
fix(SidePanel): resolves properly removing handle resize call

### PR DESCRIPTION
**Pull request by mannycarrera4**
## Summary

Resolves: https://github.com/Workday/canvas-kit/issues/73

## Checklist

- [x] branch has been rebased on the latest master commit
- [x] code has been documented and, if applicable, usage described in README.md

### CODE REVIEW

File: modules/side-panel/react/stories/stories.tsx [hunk 1]
```diff
@@ -146,7 +146,7 @@ class SidePanelWrapper extends React.Component<{}, SidePanelState> {
   };
 
   private handleBreakpoint = (open: boolean) => {
-    console.warn('called');
+    console.warn('handle breakpoint called');
```
Comment (mannycarrera4) [inline modules/side-panel/react/stories/stories.tsx]: I just added a more description console.

File: modules/side-panel/react/lib/SidePanel.tsx [hunk 2]
```diff
@@ -130,15 +130,20 @@ export default class SidePanel extends React.Component<SidePanelProps, SidePanel
     sidePanelBackgroundColor: SidePanelBackgroundColor.White,
   };
 
+  constructor(props: SidePanelProps) {
+    super(props);
+    this.handleResize = throttle(this.handleResize.bind(this), 150);
```
Comment (mannycarrera4) [inline modules/side-panel/react/lib/SidePanel.tsx]: as per @lychyi comment in: https://github.com/Workday/canvas-kit/pull/74

**Comment by lychyi** [comment]
@mannycarrera4 I think you'll need to rebase this off of the latest or fix your conflicts here before Travis builds

File: modules/side-panel/react/stories/stories.tsx (hunk 1 above)
Comment (NicholasBoll) [reply modules/side-panel/react/stories/stories.tsx]: I've seen some stories use `@storybook/addon-actions`. Do we have consensus on this? I don't know what this means for Component Story Format: https://medium.com/storybookjs/component-story-format-66f4c32366df

I think it is fine to leave this as a `console.warn` and leave the discussion for a bigger decision where we change stories to be consistent.

**Pull request merged**